Formula/wine.rb

```diff
@@ -57,13 +44,20 @@ def require_universal_deps?
   depends_on "gnutls"
   depends_on "libgsm" => :optional
 
-  # Patch to fix texture compression issues. Still relevant on 1.8.
+  # Patch to fix texture compression issues. Still relevant on 2.0.
   # https://bugs.winehq.org/show_bug.cgi?id=14939
   patch do
     url "https://bugs.winehq.org/attachment.cgi?id=52384"
     sha256 "30766403f5064a115f61de8cacba1defddffe2dd898b59557956400470adc699"
   end
 
+  # Patch to fix screen-flickering issues. Still relevant on 2.0.
+  # https://bugs.winehq.org/show_bug.cgi?id=34166
+  patch do
+    url "https://bugs.winehq.org/attachment.cgi?id=55968"
+    sha256 "1b5086798ce6dc959b3cbb8f343ee236ae06c7910e4bbae7d9fde3f162f03a79"
+  end
```

Comment:
If these patches aren't getting merged upstream we should strongly consider removing them. Any information on why upstream won't merge them?